influxdb_connector/library.yaml

```diff
@@ -0,0 +1,70 @@
+---
+
+title: influxdb Dataset Connector # required on creation, a slug will be generated from the title? -> inference-library
+version: 1.0.3 # required
+description: "Download datasets from influxdb" # optional
+icon: python # optional
+
+# Optional properties for crediting authors
+author: "cnvrg"
+author_email: "info@cnvrg.io"
+
+# At least one tag [inference, training, data] must be present
+tags:
+  - training
+  - connector
+  - data
+
+# Available only for training (exec tasks) libraries only
+command: python3 influxdb_connector.py
+
+# These will replace the "compute" argument, values must be >= 0:
+# We need to add requirement for distributed pods
+requirements:
+  cpu: 2
+  memory: 2
+  gpu: 0
+  hpu: 0
+
+# This will replace the "docker_images" argument (?) need to work in cnvrg as well
+image: python:3.8
```

Comment:
I would recommend pulling an image having the exact python version you use locally (for example: python:3.8.xx). If this doesn't work, you can always roll back to python:3.8.

Reply:
Updated to python:3.8.10 to match my local version

Reply:
This had issue importing cnvrgv2 so I am reverting back to python:3.8 image